frontend/src/metabase/admin/settings/app/components/SettingsEditor/SettingsEditor.styled.tsx

```diff
@@ -0,0 +1,11 @@
+import styled from "@emotion/styled";
+import { color } from "metabase/lib/colors";
+
+export const NewVersionIndicator = styled.span`
+  padding: 0.25rem 0.5rem 0.25rem 0.5rem;
```

Comment:
```suggestion
  padding: 0.25rem 0.5rem;
```

Actually, shouldn't we be using `Box` from Mantine?

Related comment: https://github.com/metabase/metabase/pull/32331#discussion_r1261252233